tests/cli/ErrorHandlerCest.php

```diff
@@ -0,0 +1,13 @@
+<?php
+class ErrorHandlerCest
+{
+    /**
+     * @param CliGuy $I
+     */
+    public function earlyExitWarnsTheUser(\CliGuy $I)
+    {
+        $I->executeFailCommand('run -c tests/data/first_test_exits');
+
+        $I->seeInShellOutput('COMMAND DID NOT FINISH PROPERLY');
```

Comment:
I think that it would be useful to test exit code using [seeResultCodeIs](https://codeception.com/docs/modules/Cli#seeResultCodeIs)

Reply:
Didn't know this, nice, implemented, thanks for the feedback